lib/webapi/webhandler.go

```diff
@@ -444,13 +444,30 @@ func (h *WebHandler) notFound() http.HandlerFunc {
 	}
 }
 
+func noOperationHandler() func(w http.ResponseWriter, r *http.Request, p httprouter.Params) {
+	return func(w http.ResponseWriter, r *http.Request, p httprouter.Params) {
+		log.WithFields(log.Fields{"urlPath": r.URL.Path, "method": r.Method,
+			"remoteAddr": r.RemoteAddr}).Warn("Not handling this request because the UI is disabled.")
+
+		replyError(w, "Web UI is disabled", http.StatusBadRequest)
```

Comment:
Nit: I'm not sure if BadRequest is the best error as I believe it's meant to imply a client error. what about Forbidden?